app/models/user_lesson.rb

```diff
@@ -0,0 +1,4 @@
+class UserLesson < ApplicationRecord
+  belongs_to :lesson
+  belongs_to :user
+end
```

Comment:
I don't think this is necessary if we specify the join's through  has_and_belongs_to_many  associations in both models... rails should namespace intuitively.